addon/components/code-inline.js

```diff
@@ -1,27 +1,7 @@
-/* global Prism */
-import Ember from 'ember';
-const {
-  Component,
-  computed
-} = Ember;
+import CodeBase from './code-base';
 
-export default Component.extend({
-  tagName:           'code',
-  classNames:        ['code-inline'],
-  classNameBindings: ['languageClass'],
-
-  inline:   true,
-  language: 'markup',
-
-  languageClass: computed('language', function(){
-    return 'language-' + this.get('language');
-  }),
-
-  getElement(){
-    return this.$()[0];
-  },
-
-  didInsertElement(){
-    Prism.highlightElement(this.getElement());
-  }
+export default CodeBase.extend({
+  tagName: 'code',
+  classNames: ['code-inline'],
```

Comment:
I wonder if it wouldn't be a good idea to approach this with BEM-style scoping. Something like `ember-prism__code-inline`? Then you could always pass a `class` into it as an end user. Thoughts?

Reply:
You can still pass `class` in and it'll merge the two.

Reply:
Sure, but it would conflict with any `.code-inline` styling done in the app. Scoping reduces the (granted, already small) likelihood of collision.

Reply:
We could scope, but we actually could probably just remove the classes completely. Not sure that we use them for anything important.